Sources/Microcharts/ChartEntry.cs

```diff
@@ -55,6 +55,11 @@ public ChartEntry(float value)
         /// <value>The color of the text.</value>
         public SKColor TextColor { get; set; } = SKColors.Gray;
 
+        /// <summary>
+        /// Field not used for visualisation. Can be used to store any data that could be associated with the Entry.
+        /// </summary>
+        public object EntryData { get; set; }
```

Comment:
Is the intention of this property similar to [Tag](https://docs.microsoft.com/bg-bg/dotnet/api/system.windows.frameworkelement.tag?view=netframework-4.7.2) ? If yes, it might be good idea to rename it.

Reply:
It is basically the same as a Tag, yes, but I positively hate that naming. To me, it doesn't properly convey that it is a place to store data about the element. If it is deemed to be better to follow the convention, I will rename it, but I'd rather not as I feel EntryData is much clearer about the intention of the property.